salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -30,6 +30,11 @@ Upgrade etcd cluster:
   {%- set kubeconfig = "/etc/kubernetes/admin.conf" %}
   {%- set context = "kubernetes-admin@kubernetes" %}
 
+Refresh the package repository for {{ node }}:
+  salt.function:
+    - tgt: {{ node }}
+    - name: pkg.refresh_db
+
```

Comment:
Might be relevant to have this refresh as a `require` in 
* `Set node {{ node }} version to {{ dest_version }}:`, almost certainly
* `Upgrade etcd cluster`, maybe?

Reply:
Yeah I can do that
 `Set node {{ node }} version to {{ dest_version }}:` requires `upgrade etcd cluster` so, setting `Refresh the package repository for {{ node }}:` at `Upgrade etcd cluster` should be sufficient. No??

Reply:
Should be! @TeddyAndrieux ?

Reply:
I think this state does not really require `Upgrade etcd cluster` IMO it's good to go now